documentation/modules/creating-migration-plan-2-8.adoc

```diff
@@ -1,11 +1,19 @@
 // * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="creating-migration-plan-2-6-3_{context}"]
-ifdef::provider[]
-= Creating a migration plan starting with a source provider
+[id="creating-migration-plan-2-8_{context}"]
+= Creating a migration plan
 
-You can create a migration plan based on a source provider, starting on the *Plans for virtualization* page. Note the specific options for migrations from VMware or {rhv-short} providers.
+You can create a migration plan by using the {ocp} web console to specify a source provider, the virtual machines (VMs) you want to migrate, and other plan details.
+
+[WARNING]
+====
+Virtual machines with guest initiated storage connections, such as Internet Small Computer Systems Interface (iSCSI) connections or Network File System (NFS) mounts, are not handled by {project-short} and could require additional planning before or reconfiguration after the migration.
```

Comment:
I’m not sure how to parse “not handled.”  Does this mean this type of connection fails, or perhaps {project-short} does not handle them well? The line that follows (“This is to ensure…) seems to indicate that they work but cause problems.

Since this is a warning, I’d say something like this:

Do not include virtual machines with guest initiated storage connections, such as Internet Small Computer Systems Interface (iSCSI) connections or Network File System (NFS) mounts. These  require either additional planning before a migration or reconfiguration after the migration.

Reply:
@fabiand Can you help here? Is Nunzy's suggested language OK?

Reply:
+1 on Nunzy's wording.